bot/utils/database-helper.js

```diff
@@ -51,4 +44,40 @@ module.exports.openDatabase = () => {
  */
 module.exports.getDatabase = () => {
     return database;
-};
\ No newline at end of file
+};
+
+/**
+ * Run all database migrations so that we reach the requested version.
+ * @param {string|undefined} version Which version to migrate to. If undefined, the latest version is chosen.
+ * @param {boolean} allowDown If true, allow downgrading database version. Otherwise (default), we throw an error.
+ */
+module.exports.doDatabaseMigrations = async (version = undefined, allowDown = false) => {
+    if (!version) {
+        version = migrations.latestMigration();
+    }
+
+    function exec(statements) {
```

Comment:
If exec is not already a method of the database-helper, this one should be moved out of this scope and exported as well (there will def be case where we need to run exec)

Reply:
Shouldn't it be a part of the database-crud?
While it doesn't fit the file's name directly, that one contains most other database manipulating functions. The database-helper is only used in index.js, so you'd need to export it in addition to the crud.

Reply:
Ah yes, my bad. Was indeed thinking of the database-crud